lib/merchant.rb

```diff
@@ -20,4 +22,8 @@ def items
   def invoices
     @invoices ||= @repository.invoices_by_merchant_id(@id)
   end
+
+  def one_item?
+    items.count == 1
+  end
```

Comment:
@jbranham1 how are you getting items here?

Reply:
@abreaux26 we have a method called items on the merchant!

Reply:
duhh... it was late last night and forgot to view the whole file 😅